### PR TITLE
Merge with master 2018 03 28

### PR DESCRIPTION
**Pull request by michael-wirth**

### CODE REVIEW

File: src/main/devops/data-services.yml.with.oracle [hunk 1]
```diff
@@ -0,0 +1,186 @@
+version: '3.4'
```
Comment (birom) [inline src/main/devops/data-services.yml.with.oracle]: We do not need this file any more.

File: src/main/java/ch/admin/seco/jobroom/service/dto/PasswordChangeDTO.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package ch.admin.seco.jobroom.service.dto;
```
Comment (birom) [inline src/main/java/ch/admin/seco/jobroom/service/dto/PasswordChangeDTO.java]: Is this class used somewhere?

File: src/main/webapp/app/candidate-search/services/candidate-search-request.ts [hunk 3]
```diff
@@ -32,3 +32,8 @@ export interface OccupationCode {
     type: string,
     value: number
 }
+
+export interface OccupationCode {
```
Comment (birom) [inline src/main/webapp/app/candidate-search/services/candidate-search-request.ts]: We do not need this twice.

File: src/main/devops/data-services.yml.with.oracle (hunk 1 above)
Comment (michael-wirth) [reply src/main/devops/data-services.yml.with.oracle]: yes, this was removed. it's in the batch.yml file

File: src/main/java/ch/admin/seco/jobroom/service/dto/PasswordChangeDTO.java (hunk 2 above)
Comment (michael-wirth) [reply src/main/java/ch/admin/seco/jobroom/service/dto/PasswordChangeDTO.java]: typical Merge issue. the class has been deleted

File: src/main/webapp/app/candidate-search/services/candidate-search-request.ts (hunk 3 above)
Comment (michael-wirth) [reply src/main/webapp/app/candidate-search/services/candidate-search-request.ts]: strange merge strategie... -> fixed

File: src/main/devops/infr_jobroom-proxy.config [hunk 4]
```diff
@@ -0,0 +1,266 @@
+[
```
Comment (michael-wirth) [inline src/main/devops/infr_jobroom-proxy.config]: delete

File: src/main/devops/nginx-default.conf [hunk 5]
```diff
@@ -0,0 +1,30 @@
+server {
```
Comment (michael-wirth) [inline src/main/devops/nginx-default.conf]: delete

File: src/main/webapp/app/candidate-search/state-management/state/candidate-search.state.ts [hunk 6]
```diff
@@ -32,11 +33,11 @@ export interface CandidateSearchFilter {
     residence?: Array<Canton | string>;
     availability?: Availability;
     workload?: [number, number];
-    workForm?: WorkForm;
-    degree?: Degree;
-    graduation?: Graduation;
-    drivingLicenceCategory?: DrivingLicenceCategory;
-    languageSkills?: Array<LanguageSkill>;
+    workForm?: WorkForm,
```
Comment (michael-wirth) [inline src/main/webapp/app/candidate-search/state-management/state/candidate-search.state.ts]: replace with ;

**Pull request merged**